atom/src/delattrbehavior.cpp

```diff
@@ -155,17 +155,7 @@ _mangled_property_handler( Member* member, CAtom* atom )
     cppy::ptr name( PyUnicode_FromFormat( "_del_%s", suffix ) );
     if( !name )
         return -1;
-    cppy::ptr callable( PyObject_GetAttr( pyobject_cast( atom ), name.get() ) );
-    if( !callable )
-    {
-        if( PyErr_ExceptionMatches( PyExc_AttributeError ) )
-            PyErr_SetString( PyExc_AttributeError, "can't delete attribute" );
```

Comment:
Losing this means we lose on the quality of the error message. It is sad we cannot preserve it while improving performance.
Would creating the method and then calling it with a better PyObject_Call make it much worse than the current solution.